pkg/kafka/consumer/config.go

```diff
@@ -15,21 +15,18 @@
 package consumer
 
 import (
-	"io"
+	"fmt"
 	"time"
 
 	"github.com/Shopify/sarama"
-	cluster "github.com/bsm/sarama-cluster"
 	"go.uber.org/zap"
 
 	"github.com/jaegertracing/jaeger/pkg/kafka/auth"
 )
 
 // Consumer is an interface to features of Sarama that are necessary for the consumer
 type Consumer interface {
-	Partitions() <-chan cluster.PartitionConsumer
-	MarkPartitionOffset(topic string, partition int32, offset int64, metadata string)
-	io.Closer
+	sarama.ConsumerGroup
```

Comment:
I think this defeats the purpose of this interface of being a minimal subset

Reply:
see  link:
https://github.com/jaegertracing/jaeger/pull/4294/commits/e74b66b49c4368f43ca5de5c6bc9c40562f062fe